pandas/core/window/doc.py

```diff
@@ -24,10 +24,10 @@ def create_section_header(header: str) -> str:
 
 template_see_also = dedent(
     """
-    pandas.Series.{window_method} : Calling {window_method} with Series data.
-    pandas.DataFrame.{window_method} : Calling {window_method} with DataFrames.
-    pandas.Series.{agg_method} : Aggregating {agg_method} for Series.
-    pandas.DataFrame.{agg_method} : Aggregating {agg_method} for DataFrame.\n
+    Series.{window_method} : Calling {window_method} with Series data.
+    DataFrame.{window_method} : Calling {window_method} with DataFrames.
+    Series.{agg_method} : Aggregating {agg_method} for Series.
+    DataFrame.{agg_method} : Aggregating {agg_method} for DataFrame.\n
```

Comment:
why are we modifying these? looks like the links work in the latest release:

https://pandas.pydata.org/docs/reference/api/pandas.core.window.rolling.Rolling.apply.html

but they don't work here

https://pandas.pydata.org/preview/53948/docs/reference/api/pandas.core.window.rolling.Rolling.apply.html#pandas.core.window.rolling.Rolling.apply

Reply:
I'll put them back. It was to get rid of these errors after running `validate_docstirngs.py`:
```
        pandas.Series.rolling in `See Also` section does not need `pandas` prefix, use Series.rolling instead.
        pandas.DataFrame.rolling in `See Also` section does not need `pandas` prefix, use DataFrame.rolling instead.
        pandas.Series.max in `See Also` section does not need `pandas` prefix, use Series.max instead.
        pandas.DataFrame.max in `See Also` section does not need `pandas` prefix, use DataFrame.max instead.
```
But I did forget to see the preview again, sorry.

Reply:
thanks! I'll check again once the new preview renders. this may be a bug in `validate_docstrings` unfortunately